versions/1.0/relative-time/index.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: default
+title: "Relative Time"
+version: 1.0
+---
+
+# <a href="#rel-time-introduction" id="rel-time-introduction" class="headerlink"></a> Relative Time Introduction
+
+Some APIs need to provide the ability to query data by using a relative datetime range. Some use cases are when returning analytics or report data. 
+
+# <a href="#rel-time-conformance" id="rel-time-conformance" class="headerlink"></a> Conformance
+
+The key words may, must, must not, recommended, should, and should not are to be interpreted as described in [[RFC2119](http://tools.ietf.org/html/rfc2119)].
+
+# <a href="#rel-time-conventions" id="rel-time-conventions" class="headerlink"></a> Conventions
+
+## <a href="#rel-time-conventions-uri" id="rel-time-conventions-uri" class="headerlink"></a> URI
+
+Anywhere a **URI** is specified, it **MUST** follow the rules defined in the [Hyperion]({{site.url}}/versions/{{site.latest_version}}) specification.
+
+## <a href="#rel-time-conventions-query-params" id="rel-time-conventions-query-params" class="headerlink"></a> Query Params
+
+APIs **MAY** have the following query string parameters:
+
+* `start` accepts a string value with the start date as datetime format.
+
+* `end` accepts a string value with the end date as datetime format.
+
+If _no_ query parameters are specified, the API **MUST** return a response with a default value for both fields.
+
+## <a href="#rel-time-reflected-date" id="rel-time-reflected-date" class="headerlink"></a> Reflected date
+
+To ensure that the client's request was properly executed, APIs **MUST** return the `start` and `end` datetimes as reflected values in the response body.
+
+APIs **MUST** return the datetime value following [ISO 8601](https://www.w3.org/TR/NOTE-datetime) standard as defined in the [Hyperion]({{site.url}}/versions/{{site.latest_version}}) specification.
+
+> Note: The actual properties to store those values can be API specific.
+
+Example of the `start` and `end` time range _reflected_ in the response body.
+
+`https://api.vdms.io/analytics/v1/time-series?start=now-1M`
+
+```json
+{
+    "@id": "/analytics/v1/time-series",
+    "@type": "TimeSeries",
+    "start": "2018-05-18T21:43:25Z",
+    "end": "2018-06-18T21:43:25Z",
+    ...
+}
+```
+
+# <a href="#rel-time-datetime" id="rel-time-datetime" class="headerlink"></a> Datetime Format
+
+Accepted values for `start` and `end` query string parameters **MUST** use the `now` keyword with optional modifiers.
```

Comment:
sorry I must've tuned out when we dropped the ability to query using explicit datetimes. I'm not advocating for a mixture of both in the same param, but seems like having the ability to use either/or would make sense.

e.g. start=2018-05-18T21:43:25Z&end=now